htdocs/class/xoopsform/renderer/XoopsFormRendererTailwind.php

```
@@ -0,0 +1,1116 @@
+                    . '</label>&nbsp;';
+            }
+            $ret .= $this->renderElementHtml($ele) . NWLINE;
+            if (!$isVertical) {
+                $ret .= '</span>';
+            }
+            ++$count;
+        }
+        $ret .= '</div>';
+
+        return $ret;
+    }
```

PHPMD (2.15.0)

[warning] The method renderFormElementTray() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)